### DISCUSSION TITLE
Help me out please

### DISCUSSION THREAD

**Discussion by blankenshit88** [CISOfy/lynis#1488] [Q&A]
So i am a new kali linux user and a lot of it is super confusing. But I like it. Can anyone break down into laments terms how i fix all the issues that lynis has detected in my system? Im talking a _simple_ step by step guide i guess for a new kali linux user.

**Comment by mboelen** [comment]
Hi @blankenshit88,

That is something we probably can't. Lynis is a tool meant to help you find flaw or improvements in your configuration. It is up to you as the system administrator to decide what you want to implement, how you do that, and when you do that. Every system is different.

I suggest reading the controls, check the code (shell script), and start reading. If you are new to Linux security, I suggest the [Linux Audit blog](https://linux-audit.com/) where I post both articles for beginners and specialists.
